platform/logger_no_rtos.c

```diff
@@ -0,0 +1,87 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// In case of no rtos, prints from ISR mode will be discarded
+#if !defined(MBED_CONF_RTOS_PRESENT) || defined(NDEBUG)
+
+#include "platform/mbed_logger.h"
+#include "platform/mbed_interface.h"
+#include "platform/mbed_critical.h"
+
+#ifdef __cplusplus
+extern "C" {
```

Comment:
Looking if there is any header file where `mbed_logging_start` would be declared and documented, havent found any here - what is the intention here?

Reply:
I didn't see documentation/ separate declaration of any functions in mbed_boot.c